root/etc/yum.repos.d/NsReleaseLock.repo

```diff
@@ -40,3 +40,21 @@ repo_gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
 enabled = 0
 
+[ce-sclo-sclo]
```

Comment:
This hunk is never applied to running systems because RPM puts it into `/etc/yum.repos.d/NsReleaseLock.repo.rpmnew`.

As result, if the locked origin is set, SCLo repositories may appear disabled from the command line!